js/reference.js

```diff
@@ -2900,7 +2899,12 @@ var ERMrest = (function(module) {
                         } else {
                             values[i] = column.formatPresentation(keyValues[column.name], { formattedValues: keyValues , context: this._pageRef._context });
                             // If the column type is json or jsonB we will send the templated string with <pre> tag
-                            if (column.type.name === "gene_sequence"|| column.type.name === "json" || column.type.name === "jsonb") {
+                            if(column.type.name === "json" || column.type.name === "jsonb"){
```

Comment:
Also add a check for isHTML. We only want to append `pre` tags when output is not html.